### DISCUSSION TITLE
Is there a way to force borders on windows?

### DISCUSSION THREAD

**Discussion by mfedatto** [i3/i3#6093] [Q&A]
I'm new to i3, been using it for a couple of days now. I've searched for it, but I may not be using the proper terms. Is there a way to force all windows to have the i3 border? At the exact moment, I have an Alacritty, an Edge, a VS Code, and a WhatsApp Edge web app. All of them have borders, as expected, except the WhatsApp web app, which has no borders at all. It bugs me because it is an Edge app, after all.

**Comment by orestisfl** [comment]
Put
```
for_window [all] border normal 5
```

In your config
